src/ray/protobuf/gcs.proto

```diff
@@ -212,6 +212,10 @@ message TaskStateUpdate {
   optional int64 finished_ts = 6;
   // Timestamp when status changes to FAILED.
   optional int64 failed_ts = 7;
+  // The size of input objects ray.get is called on.
+  optional uint64 input_size_bytes = 8;
```

Comment:
why is one uint64 the other int64 btw?

Reply:
I found the return value of GetSize is uint64_t whereas object_size is int64. I just kept the same type (do you know if the conversion will be applied reliably?)

Reply:
Interesting - does `object_size` go negative in any scenarios?

Reply:
let's use int64_t per https://docs.google.com/document/d/1i-UMmEYtb0iFYqc68mnJErzfhf2Q3PKHBCsuscOy5ek/edit#

Reply:
> Interesting - does object_size go negative in any scenarios?

My understanding is that it shouldn't for this reply. But there are parts that object size can be -1 (e.g., reference counter data structure). For this reply, we always guaranteed that the object is created, so it shouldn't be negative

Reply:
what's going to happen if we move uint64 to int64? We calculate the object size using uint64 (cuz that's the output type of GetSize()). Is it normally okay?